spec/api-browser-window-spec.js

```diff
@@ -1539,6 +1539,23 @@ describe('BrowserWindow module', () => {
           done()
         })
       })
+
+      it('validate process.env access in sandbox renderer', (done) => {
+        ipcMain.once('answer', function (event, test) {
+          assert.equal(test, 'foo')
+          done()
+        })
+        process.env.main = 'foo'
```

Comment:
>  Any issue you can spot in the test?

This is the issue: The tests are run in an electron renderer and you are assigning to the renderer environment. To assign to the browser process environment, use `remote.process.env.main = 'foo'`